tasks/cdn-cli.js

```diff
@@ -0,0 +1,188 @@
+import chalk from 'chalk';
+import textTable from 'text-table';
+import { toCdnEntryName } from './cdn-entry-name.js';
+import { CDN_ENVIRONMENTS } from './cdn-environments.js';
+import { CdnManager } from './cdn-manager.js';
+import { NO_CACHE } from './cellar-client.js';
+import { getCurrentBranch } from './git-utils.js';
+
+async function run () {
+  const args = getArgs();
+
+  if (args.command === '-h' || args.command === 'help') {
+    return help();
+  }
+
+  const cdnEnvironment = getCdnEnvironment(args);
+  switch (args.command) {
+    case 'get':
+      return get(cdnEnvironment, await getCdnEntryName(cdnEnvironment, args));
+    case 'list':
+      return list(cdnEnvironment);
+    case 'publish':
+      return publish(cdnEnvironment, await getCdnEntryName(cdnEnvironment, args));
+    case 'delete':
+      return remove(cdnEnvironment, await getCdnEntryName(cdnEnvironment, args));
+    case 'ui':
+      return publishUi(cdnEnvironment);
+  }
+
+  console.log(chalk.red.bold('[!] Unknown command'));
+  printAvailableCommands();
```

Comment:
In the `tasks/preview.js`, an unknown command throws an error and the `run().catch((e) => {})` triggers and `process.exit(1)`.

suggestion: can we do the same here?

Reply:
I agree with here we need to exit 1. but I won't do it with a throw error since the error message is not exactly the same as when an error occurs during command execution.